<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>561</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QLCDNumber" name="lcdPlayer1">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QLCDNumber" name="lcdPlayer2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>141</width>
      <height>50</height>
     </rect>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>461</width>
      <height>441</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <layout class="QGridLayout" name="gameGrid" rowminimumheight="0">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <property name="verticalSpacing">
       <number>6</number>
      </property>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuArquivo">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="acNewGame"/>
    <addaction name="acQuit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="acAboutQT"/>
   </widget>
   <addaction name="menuArquivo"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="acQuit">
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
  <action name="acAboutQT">
   <property name="text">
    <string>About QT</string>
   </property>
  </action>
  <action name="acNewGame">
   <property name="text">
    <string>New Game</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
